### PR TITLE
Add inline generate command to "Defining a component"

### PR DESCRIPTION
**Pull request by joostdevries**
Seems to fit here
Refs #410

### CODE REVIEW

File: source/components/defining-a-component.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ create a `components/blog-post` template.
 **Note:** Components must have at least one dash in their name. So `blog-post` is an acceptable name, so is `audio-player-controls`, but `post` is not. This prevents clashes with current or future HTML element names, and
 ensures Ember detects the components automatically.
 
+```bash
+ember generate component blog-post
+```
+
 A sample component template would look like this:
```
Comment (michaelrkn) [inline source/components/defining-a-component.md]: I think put the generation after this introductory sentence.

Comment (locks) [inline source/components/defining-a-component.md]: We should decide on https://github.com/emberjs/guides/issues/519 and introduce generators at the top of the guides and be done with it.

Comment (michaelrkn) [inline source/components/defining-a-component.md]: I think @joostdevries should make this quick change so we can get it merged in - there are already several other places his addition of generators have been merged in, and if we go forward with #519 we can replace them all.

Comment (locks) [inline source/components/defining-a-component.md]: :+1:

**Comment by joostdevries** [comment]
done.

**Comment by michaelrkn** [comment]
awesome. final thing - can you squash these commits? thanks!

**Comment by joostdevries** [comment]
:+1:

**Pull request merged**